app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :password, presence: true, length: { minimum: 8 }
+  validates :role, presence: true
+  validates :email, presence: true, length: { minimum: 8 }
```

Comment:
That's fine for test tasks. But if I saw it in a project, I'd mention that this is a bad practice to squash different layers (DB Gateways and Business Layer) in a one place
![image](https://user-images.githubusercontent.com/12776494/197275293-521b5c9e-73ef-4e65-aacf-14af86252c0d.png)